<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationDialog</class>
 <widget class="QDialog" name="TransformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformation</string>
  </property>
  <layout class="QVBoxLayout" name="transformationDialogLayout">
   <item>
    <layout class="QHBoxLayout" name="recordInfoLayout">
     <item>
      <widget class="QLabel" name="labelRecordType">
       <property name="text">
        <string>Record type: &lt;i&gt;None&lt;/i&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string>Time: &lt;i&gt;00:00&lt;/i&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="transformationBox">
     <property name="title">
      <string>Transformations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="optionLayout">
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkChangeSpeed">
          <property name="text">
           <string>Change speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkKeyboardOnly">
          <property name="text">
           <string>Make keyboard only</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkMouseOnly">
          <property name="text">
           <string>Make mouse only</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxSpeed">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkRemoveWaitTime">
          <property name="text">
           <string>Remove wait time before first action</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="buttonOK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
